guide/14-deep-learning/FCN_MLModel_guide-final_draft.ipynb

```diff
@@ -0,0 +1,501 @@
+{
```

Comment:
<p>(1) The datefield if it is mentioned, <s>are</s><strong> is </strong>always split into its day, month, year component</p><p>(2) Suppose input features <s>can have</s> <strong>has </strong>fields A, B, C, D, E, F, <strong>and X</strong> where the fields A, B <strong>and C</strong> are used as the variables to predict X.</p><p>(3) If the user wants to split field D and add the components as the predictors then <strong>it</strong> is mentioned as date_field=D</p><p>(4) If the user <s>don't</s> <strong>doesn't </strong>to use field D</p><p>(5) Again <strong>if</strong> the user wants to use <strong>the</strong> date field without splitting into its day, month, year component then it could just <strong>be</strong> passed in the list</p><p>(6) only if it split and added as <strong>a</strong> predictor, other than special cases</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>